docs/release-notes/v2.0.0.md

```diff
@@ -0,0 +1,24 @@
+## API Changes and Discord Integration
+
+**Breaking change**: the previous input parameter `SLACK_TOKEN` and `CHANNEL`
+have been deprecated and cannot be used anymore. Use `SLACK_WEBHOOK_URL` and
+`SLACK_CHANNEL` instead.
+
+`howl` can now be used as GitHub Action with `with` inputs:
+
+```yaml
+  howl-on-failure:
+    steps:
+      - uses: foxygoat/howl@v2
+        with:
+          slack_webhook_url: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
if you do switch to camel case, don't forget to update these release notes

Reply:
ok, done.